string ids_command
---
bool transition